GOVERNANCE.md

```diff
@@ -11,21 +11,36 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "organization voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+All the following conditions must be satisfied to approve the formal votes.
+
+- No effective objection ballot: See Vetoes, below.
+- At least effective __2-organization__ affirmative vote: See Organization Vote, below.
+- At least effective __3-maintainers__ affirmative vote.
+- At least __2-week__ for voting.
```

Comment:
To change the voting system, we must follow the CNCF's election policy:
https://github.com/cncf/foundation/blob/main/maintainers-election-policy.md

>    Each eligible project maintainer will receive one vote. In order to participate in the election process, maintainers will need to add themselves to the list of active voters. This will occur at the same time as the nomination process and last at least two weeks.

We must follow the Voting System from **CNCF** not Apache.
But, I agree with your suggestion. It's acceptable change. :+1:

Reply:
> We must follow the Voting System from CNCF not Apache.

Oh! I see! Sorry for missing that point.

> But, I agree with your suggestion. It's acceptable change. 👍

Thanks for your review from CNCF's point of view!

Reply:
However, CNCF's guideline declares project maintainers as follows:

> These project maintainer lists will be determined by the public lists at maintainers.cncf.io.

Have you registered and received an email from maintainers.cncf.io, @daipom?
If not, would you mind if you registered the mailing list?

Reply:
Ah, we should post a pull request to https://github.com/cncf/foundation like this: https://github.com/cncf/foundation/pull/504
Thanks for notifying it.

Reply:
Oh, I didn't know it! Thanks!

Reply:
https://github.com/cncf/foundation/pull/553

Reply:
Added: https://maintainers.cncf.io

Reply:
Thanks!